threatgrid/ctia
# bulk patch routes

> **Epic** #https://github.com/advthreat/iroh/issues/2249

The PR implements bulk patch routes and most of the logic of bulk update.

<a name="qa">[§](#qa)</a> QA
============================


1. create 3 incidents and note their ids 
2. `PATCH /ctia/bulk`
``` javascript
{
  "incidents":
    [{"id": "incident-id-1", "source": "patched"},
     {"id": "incident-id-2", "source": "patched"},
     {"id": "incident-id-2", "source": "patched"}]
}
```
3. bulk export these patched incidents and check they are actually updated
4. check that update events are created for these incidents.

<a name="release-notes">[§](#release-notes)</a> Release Notes
=============================================================



```
intern: bulk patch route in CTIA
```
